plotly/tests/test_optional/test_figure_factory.py

```diff
@@ -672,3 +678,42 @@ def test_valid_endpts(self):
                                 tls.FigureFactory.create_scatterplotmatrix,
                                 df, use_theme=True, index='a',
                                 palette='Blues', endpts=[2, 1])
+
+    def test_scatter_plot_matrix_kwargs(self):
+
+        # check if test scatter plot matrix matches with
+        # the expected output
+
+        df = pd.DataFrame([[2, 'Apple'], [6, 'Pear'],
+                          [-15, 'Apple'], [5, 'Pear'],
+                          [-2, 'Apple'], [0, 'Apple']],
+                          columns=['Numbers', 'Fruit'])
+
+        test_scatter_plot_matrix = tls.FigureFactory.create_scatterplotmatrix(df, index='Fruit', endpts=[-10, -1],
+                                                                              diag='histogram', height=1000, width=1000,
+                                                                              size=13, title='Scatterplot Matrix',
+                                                                              use_theme=True, palette='YlOrRd',
+                                                                              marker=dict(symbol=136))
```

Comment:
PEP8 for line length here, you can just wrap this all. It might seem fussy, but it's super helpful to follow these style rules!